NodeResourceTemplate.yml

```diff
@@ -0,0 +1,30 @@
+# TOP LEVEL TEMPLATE:
+# Can only be used to run the single NodeTemplate pipeline.
+# Why: Resources can only be defined once.
+#      This determines the resources available to all "jobs" and "steps" no matter which templates are loaded after this
+#      If resources is ever defined again, it'll break so badly that the pipeline won't even parse
+
+resources:
+  repositories:
+    - repository: JenkinsSdkSetupScripts
+      type: github
+      endpoint: GitHub_PlayFab
+      name: PlayFab/JenkinsSdkSetupScripts
+    - repository: API_Specs
+      type: github
+      endpoint: GitHub_PlayFab
+      name: PlayFab/API_Specs
+    - repository: SdkGenerator
+      type: github
+      endpoint: GitHub_PlayFab
+      name: PlayFab/SdkGenerator
+    - repository: NodeSDK
+      endpoint: GitHub_PlayFab
+      type: github
+      name: PlayFab/NodeSDK
```

Comment:
NodeSDK repo defined here is (almost) equivalent to "self" and if "self" is used properly, it doesn't need to be redefined. Self is better for various reasons.

Reply:
(delete this NodeSDK repo defn, and follow the comment to use "self" below)